dbms/src/IO/Endian.h

```diff
@@ -18,6 +18,11 @@
 
 namespace DB
 {
+inline constexpr bool isLittleEndian()
```

Comment:
is `constexpr` means it's decided at comiler time?

Reply:
Yes

Reply:
however, ref to doc, I found `native` is determined at execution environment

Reply:
The value of order::native shall be:

equal to order::big if the **execution environment** is big endian, otherwise

equal to order::little if the **execution environment** is little endian, otherwise

unequal to both order::little and order::big.

Reply:
I'll check it.

Reply:
From contrib/boost/boost/endian/detail/order.hpp, it looks like the native value can be initialized to either big or little from compilation environments.

Reply:
ok

Reply:
Oh, there is a third possibility: __ORDER_PDP_ENDIAN__, seems mixture of little and big, which I think we'd not support now.